interface/ceed-preconditioning.c

```diff
@@ -187,7 +187,8 @@ static int CeedOperatorCreateFallback(CeedOperator op) {
 
   @ref Developer
 **/
-static inline int CeedSingleOperatorAssembleAddDiagonal_Core(CeedOperator op, CeedRequest *request, const bool is_point_block, CeedVector assembled) {
+static inline int CeedSingleOperatorLinearAssembleAddDiagonal_Mesh(CeedOperator op, CeedRequest *request, const bool is_point_block,
+                                                                   CeedVector assembled) {
```

Comment:
I don't like `Mesh` as a name for "not AtPoints" for some reason, but I also can't think of a better name, nor am I familiar with the relationship between AtPoints and the "normal" operators.

Reply:
Yeah, I don't love it, but I don't have a better name either

Reply:
We can always change it later if we think of something better.